dependencies/pom.xml

```diff
@@ -104,8 +104,8 @@
         <version.lib.microprofile-metrics-api>2.3.2</version.lib.microprofile-metrics-api>
         <version.lib.microprofile-openapi-api>1.1.2</version.lib.microprofile-openapi-api>
         <version.lib.microprofile-reactive-messaging-api>1.0</version.lib.microprofile-reactive-messaging-api>
-        <version.lib.microprofile-reactive-streams-operators-api>1.0.1</version.lib.microprofile-reactive-streams-operators-api>
-        <version.lib.microprofile-reactive-streams-operators-core>1.0.1</version.lib.microprofile-reactive-streams-operators-core>
+        <version.lib.microprofile-reactive-streams-operators-api>2.0</version.lib.microprofile-reactive-streams-operators-api>
```

Comment:
Reactive streams messaging depends on operators. Isn't there some conflict (as it this is a major version)?

Reply:
Its major version but there was no significant changes from messaging side I am aware of:
https://download.eclipse.org/microprofile/microprofile-reactive-streams-operators-2.0/microprofile-reactive-streams-operators-spec-2.0.html#release_notes_20

We can delay merging until messaging upgrade is ready, but I don't believe its a blocker